rhv_checks.py

```diff
@@ -338,6 +338,53 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, warning, all_items = [], [], [], []
+    # Get all the hosts with details.
+    hosts = system.api.system_service().hosts_service().list(all_content=True)
+
+    for host in hosts:
+        host_info = {
+            "Configured": host.hosted_engine.configured,
+            "Active": host.hosted_engine.active,
+            "local_maintenance": host.hosted_engine.local_maintenance,
+            "global_maintenance": host.hosted_engine.global_maintenance,
+            "Score": host.hosted_engine.score,
+        }
+        engine_value = all((host_info["Active"], host_info["Configured"]))
+        maintenance_value = any(
+            (host_info["global_maintenance"], host_info["local_maintenance"])
+        )
+        if not engine_value or maintenance_value:
+            critical.append((host.name, host_info))
+        else:
+            okay.append((host.name, host_info))
+        # Checking hosted engine non-zero scrore.
+        if host_info["Score"] == 3400:
+            okay.append((host.name, host_info["Score"]))
```

Comment:
You `append` the okay host twice, `if host_info["Score"] == 3400`and if the above else statement is satisfied.

Reply:
Done. Removed.